plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -66,6 +67,28 @@ func (a *AlwaysPullImages) Admit(ctx context.Context, attributes admission.Attri
 		return apierrors.NewBadRequest("Resource was marked with kind Pod but was unable to be converted")
 	}
 
```

Comment:
also need to apply the same fix to the Validate() method

Reply:
suggest factoring this into a `isUpdateWithNoNewImages(attributes) bool` method and calling it both from Admit() and Validate()

Reply:
fixed